### PR TITLE
Add /usr/local/bin to PATH on admin

### PR DESCRIPTION
**Pull request by jwhitlock**
The stylus and cleancss node commands are symlinked to /usr/local/bin on admin.  This allows ./manage.py collectstatic to compile stylesheets on deploys.

### CODE REVIEW

File: scripts/chief_deploy.py [hunk 1]
```diff
@@ -16,7 +16,9 @@
 
 # Setup venv path
 venv_bin_path = os.path.join(settings.VENV_DIR, 'bin')
-os.environ['PATH'] = venv_bin_path + os.pathsep + os.environ['PATH']
+os.environ['PATH'] = os.pathsep.join(
```
Comment (jezdez) [inline scripts/chief_deploy.py]: This needs to be `os.environ['PATH'] = os.pathsep.join([venv_bin_path, '/usr/local/bin', os.environ['PATH']])`, note the square brackets

**Comment by robhudson** [comment]
After jezdez suggested fix this looks good to me.

**Comment by jwhitlock** [comment]
Force push.  Converted to ``join([...])``, and added newlines to make flake8 happy.

**Comment by jezdez** [comment]
r+ after Travis

**Comment by openjck** [comment]
Travis passing :smile:

**Pull request merged**